vignettes/manual/ContributionsGuide.Rmd

```diff
@@ -8,7 +8,7 @@ All contributions must pass through a pull request (PR) process before being mer
 
 We encourage all contributors to read and comply with the following guidelines:
 
-* Before embarking on your work open a [development](link) GitHub issue to provide a brief description of your change or development, including justification and timeline. This allows us to provide feedback, and helps us plan and prepare for your PR;
+* Before embarking on your work, open a [development](link) GitHub issue to provide a brief description of your change or development, including justification and timeline. This allows us to provide feedback, and helps us plan and prepare for your PR;
```

Comment:
replace `[development](link)` with `[development](https://github.com/JGCRI/hector/issues/new?assignees=&labels=&template=development.md)` this will hyperlink to the correct page